phalcon/forms/form.zep

```diff
@@ -290,17 +290,19 @@ class Form extends Injectable implements \Countable, \Iterator
 			}
 		}
 
-		let notFailed = true,
-			messages = [];
+		let validationStatus = true;
 
+        let validation = this->getValidation();
+
+        if !(validation instanceof Validation) {
+            // Create an implicit validation
+            let validation = new Validation();
```

Comment:
Well, I think I misled you. The exclamation point has a higher priority. So could you please use:
```zephir
if !(validation instanceof Validation) {
```

But actually I think that more flexibility was to use the interface but not concrete class